bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/ChannelPool.java

```diff
@@ -66,17 +97,22 @@
    */
   public ChannelPool(List<HeaderInterceptor> headerInterceptors, ChannelFactory factory)
       throws IOException {
-    ManagedChannel channel = factory.create();
-    this.channels.set(ImmutableList.of(channel));
-    authority = channel.authority();
+    getActiveChannelCounter().inc();
     this.factory = factory;
+    final ChannelAndMetrics channelAndMetrics = createChannelAndMetrics();
+    this.channels.set(ImmutableList.of(channelAndMetrics));
+    authority = channelAndMetrics.channel.authority();
     if (headerInterceptors == null) {
       this.headerInterceptors = ImmutableList.of();
     } else {
       this.headerInterceptors = ImmutableList.copyOf(headerInterceptors);
     }
   }
 
+  private ChannelAndMetrics createChannelAndMetrics() throws IOException {
+    return new ChannelAndMetrics(factory.create());
+  }
+
   /**
    * Makes sure that the number of channels is at least as big as the specified capacity.  This
    * method is only synchornized when the pool has to be expanded.
```

Comment:
Not part of this change but mind fixing the spelling mistake in the comment? s/synchornized/synchronized/